CollectionsInterface/Exercise6/src/Club.java

```diff
@@ -0,0 +1,132 @@
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+public class Club {
+
+    private Map <UUID, Boolean> disqualifiedMembers;
+    private Map <UUID, Boolean> alreadyVoted;
+    private Map <UUID, Integer> votes;
+    private Map <UUID, ClubMember> members;
```

Comment:
```suggestion
    private Set <ClubMember> members;
```
Te ahorras el lio de crear dos collections en el metodo para agregar un nuevo clubMember